### DISCUSSION TITLE
The channel was closed: AMQP close-reason code=503, text='COMMAND_INVALID

### DISCUSSION THREAD

**Discussion by attiqeurrehman** [rabbitmq/rabbitmq-delayed-message-exchange#275] [Q&A]
Hi,

We are using the plugin with `masstransit `and we are experiencing weird behavior even with plugin enabled we are unable to use it and getting the following error:

> The channel was closed: AMQP close-reason, initiated by Peer, code=503, text='COMMAND_INVALID - unknown exchange type 'x-delayed-message'', classId=40, methodId=10 9

![Screenshot 2024-03-25 143312](https://github.com/rabbitmq/rabbitmq-delayed-message-exchange/assets/1382615/cd07e954-eedf-4ab8-ab33-b7113b41d4e8)

**Comment by lukebakken** [comment]
Hello, thanks for using RabbitMQ.

Unfortunately, there is very little information to work with here. Some questions:
* What is your Erlang version?
* What operating system and version are you using?
* How **EXACTLY** did you download and install this plugin?
* How **EXACTLY** did you enable this plugin?

**Comment by gomoripeti** [comment]
In the output of `rabbitmq-plugins list` usually there is a `*` after the `E` or `e` that indicates the enabled plugin is running. This suggests me that the cli can access the enabled plugins file but either cannot access the running broker or the plugins are not running. (ie something is fishy) Answering the questions of lukebakken might shed some light on what could be wrong.

**Comment by attiqeurrehman** [comment]
Hi,

Sorry for missing out on the information, here you go:

**What is your Erlang version?** 25.2.1
**What operating system and version are you using?**  Windows Server 2019
**How EXACTLY did you download and install this plugin?** 
Plugin was downloaded from [here](https://github.com/rabbitmq/rabbitmq-delayed-message-exchange/releases/tag/3.11.1)
**How EXACTLY did you enable this plugin?** 
Copied in the plugin folder, ran the command `rabbitmq-plugins enable rabbitmq_delayed_message_exchange`
and it showed the above status in the question.
